tests/test_rooms/test_query_rooms.py

```diff
@@ -28,7 +30,9 @@ def test_paginate_room_query(self):
         expected_response = paginated_rooms_response
         self.assertEqual(paginate_query, expected_response)
 
-    def test_query_remote_rooms(self):
+    @patch.object(CalendarApi, "calendar_list", spec=True,
+                  return_value=remote_rooms)
+    def test_query_remote_rooms(self, mocked_method):
```

Comment:
It's good practice to include docstrings in your functions and classes. It provides a convenient way of
associating documentation with Python modules, functions, classes, and methods

Reply:
I have implemented this.